### PR TITLE
Fix for black screenshots on Windows

### PR DESCRIPTION
**Pull request by mBeierl**
I was having issues running record.py on my windows 10 machine. I don't know the exact reason, but after using wx.EmptyBitmap instead of wx.Image and modifying the take_screenshot function in utils.py it worked fine.

@kevinhughes27 please test it on your machine (it was linux right?) before accepting it.

### CODE REVIEW

File: record.py [hunk 1]
```diff
@@ -125,7 +125,7 @@ def draw(self):
         # Image
         img = self.bmp.ConvertToImage()
         img = img.Rescale(320,240)
-        self.image_widget.SetBitmap( img.ConvertToBitmap() )
+        self.image_widget.SetBitmap(self.bmp)
```
Comment (kevinhughes27) [inline record.py]: I kind of want to rename this to `self.screenshot` now that I return to this code. What do you think?

**Comment by kevinhughes27** [comment]
LGTM I'll test it out and make sure it works on Linux

File: record.py (hunk 1 above)
Comment (mBeierl) [inline record.py]: yes! makes sense.

**Comment by kevinhughes27** [comment]
:crying_cat_face:  `AttributeError: 'module' object has no attribute 'MemoryDCFromDC'`

**Comment by kevinhughes27** [comment]
That sucks - there has to be a cross platform technique / wrapper for taking screenshots. Did you look at https://pypi.python.org/pypi/pyscreenshot ? I remember trying it but I can't remember why I didn't use it unfortunately.

**Comment by mBeierl** [comment]
hm, damn. 

Yes I've looked at pyscreenshot and it seems to work. It basically looks for one of the supported back-ends and uses them for basic screenshot capabilities. So at least one of the back-ends has to be installed. Haven't tried to incorporate it into the GUI though.

If you want to move away from wx completely, what GUI library would you prefer? Kivi, PyQt or something else? I have some time next week, so I could actually work on a transition from wx to another library and then use pypi/pyscreenshot.

**Comment by kevinhughes27** [comment]
I don't have a strong preference - I just want the dependencies to be as simple as possible. My guess is that Qt is probably the best option. If you have a preference then by all means run with it and pitch why its the best option :)

**Comment by mBeierl** [comment]
I've started development to port from wx to PyQt. It has screenshot capabilities built in, so no need for pyscreenshot. Will further work on it in [my fork's branch](https://github.com/mBeierl/TensorKart/tree/wip/wxToPyQt) and create a PR once it's done.

**Pull request closed without merging**